### PR TITLE
Standardise call out  boxes

### PR DESCRIPTION
**Pull request by Lsnaathorst1**
## Overview of changes
Updates to standardise the use of call out boxes across the analysts guide

## Why are these changes being made?
We were inconsistent with the use of call out boxes across the analysts guide 

## Detailed description of changes
Updated to use the inbuilt quarto call outs as were already being used in some places. Have used call outs: note, warning, important and tip. Where it seems helpful to the reader, I have included a title, as was already in use in the ADA and databricks section. 
## Issue ticket number/s and link
Standardise use of info / warning / alert boxes #71
## Checklist before requesting a review
- [X] I have checked the contributing guidelines
- [X] I have checked for and linked any relevant issues that this may resolve
- [x] I have checked that these changes build locally
- [X] I understand that if merged into main, these changes will be publicly available

### CODE REVIEW

**Review by rmbielby — CHANGES_REQUESTED** [review]

File: statistics-production/ees.qmd [hunk 1]
```diff
@@ -985,9 +986,9 @@ Use plain English and shorter sentences, aim for an average of 15-20 words per s
 
 Explain complex concepts in plain English within the text. Remember that for many of our users, confidence intervals and significant differences are complex concepts that need explaining.
 
-<div class="alert alert-dismissible alert-danger">
+::: {.callout-warning}
```
[the remark this answers is not in the document]
Comment (Lsnaathorst1) [reply statistics-production/ees.qmd]: Removed :)

**Review by rmbielby — APPROVED** [review]
